tests/test_cli/test_inputhandler/test_inputhandler.cpp

```diff
@@ -341,7 +341,7 @@ TEST(JsonFileHandler, writeOutputFile)
                             transactionWrapper.getGasPrice(), transactionWrapper.getGasLimit(),
                             transactionWrapper.getData(), transactionWrapper.getChainId(),
                             transactionWrapper.getVersion());
-    Signer signer(pemHandler.getPrivateKey());
+    Signer signer(pemHandler.getSeed());
```

Comment:
remove commented code line 298

Reply:
Indeed, it is ugly to have commented code, but is left there for future tests that are coming in the next PR. We will remove it from there